docs/getting-started/docker-network-plugin/GettingStarted.md

```diff
@@ -0,0 +1,235 @@
+
+- [Vagrant install with CoreOS](../VagrantCoreOS.md)
+- [Vagrant install with Ubuntu](../VagrantUbuntu.md)
+- [Amazon Web Services (AWS)](../AWS.md)
+- [Google Compute Engine (GCE)](../GCE.md)
+- [DigitalOcean](../DigitalOcean.md)
+
+Altenatively, you can manually configure your hosts.
+- [Manual setup](ManualSetup.md)
+
+If you have everything set up properly you should have `calicoctl` in your 
+`$PATH`, and two hosts called `calico-01` and `calico-02`.
+
+## 2. Starting Calico services
+
+Once you have your cluster up and running, start calico on all the nodes, 
+specifying the `--libnetwork` option to start the Calico network and IPAM 
+driver in a separate container.
+
+On calico-01
+
+    sudo calicoctl node --libnetwork
+
+On calico-02
+
+    sudo calicoctl node --libnetwork
+
+This will start a container on each host. Check they are running
+
+    docker ps
+
+You should see output like this on each node
+
+    vagrant@calico-01:~$ docker ps
+    CONTAINER ID        IMAGE                           COMMAND                  CREATED             STATUS              PORTS               NAMES
+    eec9ebbfb486        calico/node-libnetwork:latest   "./start.sh"             21 seconds ago      Up 19 seconds                           calico-libnetwork
+    ffe6cb403e9b        calico/node:latest              "/sbin/my_init"          21 seconds ago      Up 20 seconds                           calico-node
+
+## 3. Create the networks
+
+The Calico network driver allows you to use the `docker network` command to
+manage networks that use Calico networking.
+
+Let's go ahead and create some networks and start a few containers 
+on each host spread between these networks.
+
+> Exactly how we create these networks depends on whether we are using the
+> built-in libnetwork IPAM driver or the Calico IPAM driver.  For more details
+> on this, see the documentation for [`calicoctl node`](../../calicoctl/node.md)
+>
+> Furthermore, when running in a cloud environment we need to also set
+> `ipip` and `nat-outgoing` options.
+
+In each case we use the Calico driver `calico` for networking.  This driver is
+run as part of the calico-node in a separate calico/node-libnetwork container.
+For this worked example, we explicitly choose an IP Pool for each network 
+rather than using the default selections - this is to avoid potential conflicts
+with the default NAT IP assignment used by VirtualBox.  Depending on your
+specific environment, you may need to choose different IP Pool CIDRs.
+
+When the default IPAM driver is used, the pool CIDR and `ipip` and 
+`nat-outgoing` configuration are specified as options on the `network create`.
+
+When the Calico IPAM driver `calico` is used, it is necessary to create the 
+Calico IP Pool in advance with the appropriate configuration for `ipip` and 
+`nat-outgoing`.  The Calico IPAM driver runs alongside the Calico network
+driver within the calico/node-libnetwork container.
```

Comment:
Agree - I actually think these are redundant since we say we start the driver in (2).